static/js/notifications.js

```diff
@@ -705,25 +702,28 @@ export function reify_message_id(opts) {
 }
 
 export function register_click_handlers() {
-    $("#out-of-view-notification").on("click", ".compose_notification_narrow_by_topic", (e) => {
-        const message_id = $(e.currentTarget).data("message-id");
-        narrow.by_topic(message_id, {trigger: "compose_notification"});
-        e.stopPropagation();
-        e.preventDefault();
-    });
-    $("#out-of-view-notification").on("click", ".compose_notification_scroll_to_message", (e) => {
-        const message_id = $(e.currentTarget).data("message-id");
-        message_lists.current.select_id(message_id);
-        navigate.scroll_to_selected();
-        $("#out-of-view-notification").hide();
-        e.stopPropagation();
-        e.preventDefault();
-    });
-    $("#out-of-view-notification").on("click", ".out-of-view-notification-close", (e) => {
-        clear_compose_notifications();
-        e.stopPropagation();
-        e.preventDefault();
-    });
```

Comment:
I guess we previously weirdly closed all notifications if you clicked any of their `X` elements; I assume that's fine to remove it.

Reply:
It's removed because it's now handled by the compose-banner logic which adds a close-button-handler to all banners.